src/web/app/pages-instructor/instructor-search-page/comment-result-table/comment-result-table.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FeedbackSession, QuestionOutput } from "../../../../types/api-output";
+
+/**
+ * Data object for communication with child comment result component
+ */
+export interface SearchCommentsTable {
```

Comment:
I guess you may even use `CommentSearchResult` here

Reply:
Right, that's possible too. But i think its better to just split it into the feedback session and question. I feel that it is more readable this way and you have direct access to the session/question instead of going through another layer of data